File: lab1/main.py
l_array = [23, 4, 67, 1, 0, -20, 29, -100, 32, 99, -56]

print ("[", end= "");

for i in range(len(l_array)):
    print (l_array[i], end="");
    
    if i + 1 < len(l_array):
        print (",", end = "");
        
print("]")

File: lab1/Main.cpp
#include <cstddef>

extern void printIntArray(int aArray[], size_t aSize);

int main()
{
    int lArray[] = { 23, 4, 67, 1, 0, -20, 29, -100, 32, 99, -56 };
    size_t lSize = sizeof(lArray) / sizeof(int);

    printIntArray(lArray, lSize);

    return 0;
}